test/e2e/framework/util.go

```diff
@@ -3795,6 +3797,13 @@ type SSHResult struct {
 	Code   int
 }
 
+// NodeExec execs the given cmd on node via SSH. Note that the nodeName is an sshable name,
+// eg: the name returned by framework.GetMasterHost(). This is also not guaranteed to work across
+// cloud providers since it involves ssh.
+func NodeExec(nodeName, cmd string) (SSHResult, error) {
+	return SSH(cmd, fmt.Sprintf("%v:%v", nodeName, sshPort), TestContext.Provider)
```

Comment:
Ok. I'm open to something better. I see this type of pattern where an exported func calls an internal func, adding args to the call. Not sure if the added args are global or not. Is this a deal breaker?